dimagi/pages/templates/svg/sectors/agriculture-extension-programs/AgExtension_Icon1.html

```diff
@@ -0,0 +1,12 @@
+
+<svg width="68px" height="68px" viewBox="0 0 68 68" version="1.1" >
+    <g id="1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round">
+        <g id="Icon-3" transform="translate(7.000000, 7.000000)" stroke="#47b700" stroke-width="2">
+            <path d="M18.7,41.7 C20.8719244,39.5205441 23.8231064,38.2968832 26.9,38.3 C29.9806651,38.2777347 32.9389605,39.504345 35.1,41.7" id="Path"></path>
+            <path d="M12.8,35.7 C16.5704157,31.9382012 21.673946,29.8177203 27,29.8 C32.3353408,29.771037 37.4561139,31.8986821 41.2,35.7" id="Path"></path>
+            <path d="M6.2,29.1 C11.6810554,23.5325281 19.1875964,20.4288621 27,20.5 C34.7993917,20.495231 42.2812877,23.5887072 47.8,29.1" id="Path"></path>
+            <path d="M0,22.9 C7.16872347,15.7611119 16.8830212,11.7674562 27,11.8 C37.1077568,11.8142531 46.8049175,15.8008636 54,22.9" id="Path"></path>
+            <path d="M8.6,0 L35.6,54.2" id="Path"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
empty new line at end of svg files please. also make sure that the 2 space indent rule is also obeyed in these files